spec/lib/hubspot/company_spec.rb

```diff
@@ -179,20 +179,19 @@
     end
   end
 
-  describe "#update!" do
-    cassette "company_update"
-    let(:company){ Hubspot::Company.new(example_company_hash) }
-    let(:params){ {name: "Acme Cogs", domain: "abccogs.com"} }
-    subject{ company.update!(params) }
+  describe ".update!" do
```

Comment:
These changes:
- updates the test to remove the use of `its` (which is deprecated but still available to us as we've pulled in the `rspec-its` gem)
- updates the test to be self-reliant by creating and deleting any necessary data.

Reply:
Nice, how far is the code base from removing `rspec-its` as a dependency?

Reply:
There are about 10 spec files that use `its` heavily. It will take some time to get there.